packages/fxa-settings/src/pages/Signin/index.tsx

```diff
@@ -229,13 +230,10 @@ const Signin = ({
             getLocalizedErrorMessage(ftlMsgResolver, navError)
           );
         } else {
-          // TODO, address signIn.verified vs session.verified discrepancy
-          // currently 'verified' only checks session status, but 'verificationReason'
-          // can tell us if it's a sign up. This will be cleaned up in FXA-12454
           if (
-            !data.signIn.verified &&
-            data.signIn.verificationReason !== VerificationReasons.SIGN_UP &&
-            isWebIntegration(integration)
+            data.signIn.emailVerified &&
```

Comment:
I had to look at this a couple times because of the previous check on `verificationReason` no longer be present. I think maybe this deserves a comment as to the scenario this covers. i.e. not an account signup and session isn't verified.